양문규/Week25/소수의연속합.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from enum import Enum
+from collections import deque
+from heapq import heappush, heappop, heapify
+import math
+
+# 입력 받기
+N = int(input().strip())
+
+
+# 소수 구하기
+def get_prime_numbers():
+    nums = [1 for _ in range(N + 1)]
+    prime_nums = []
+
+    for i in range(2, N + 1):
+        for j in range(i * 2, N + 1, i):
+            nums[j] = 0
+
+    for i in range(2, len(nums)):
+        if nums[i]:
+            prime_nums.append(i)
```

Comment:
```suggestion
    for i in range(2, N + 1):
        if nums[i]:
            prime_nums.append(i)
    
        for j in range(i * 2, N + 1, i):
            nums[j] = 0
```

소수만 따로 저장할 때 이렇게 해도 될 것 같습니다.
유효한 변경이라면 반복문 루프 한 번 줄겠네요.

Reply:
오. 이러면 더 좋겠네요.
담엔 이렇게 한 번 시도해볼게요. ㅎㅎ 좋은 내용 공유 감사합니다 👍